### ISSUE TITLE
Hint: Could not find a declaration file for module './DarkPaginationNav.svelte'.

### ISSUE THREAD

**Issue by azzamsa** [TahaSh/svelte-paginate#32]
How to fix  `Hint: Could not find a declaration file for module './DarkPaginationNav.svelte'.`?

```
> pokedex@0.1.1 check /home/username/softwares/learn/pokedex-svelte
> svelte-check --tsconfig ./tsconfig.json --workspace src


====================================
Loading svelte-check in workspace: /home/username/softwares/learn/pokedex-svelte/src
Getting Svelte diagnostics...

/home/username/softwares/learn/pokedex-svelte/src/../node_modules/.pnpm/file+..+..+forks+svelte-paginate/node_modules/svelte-paginate/src/main.ts:2:42
Hint: Could not find a declaration file for module './PaginationNav.svelte'. '/home/username/softwares/learn/pokedex-svelte/node_modules/.pnpm/file+..+..+forks+svelte-paginate/node_modules/svelte-paginate/src/PaginationNav.svelte' implicitly has an 'any' type.
export { default as paginate } from './paginate'
export { default as PaginationNav } from './PaginationNav.svelte'
export { default as LightPaginationNav } from './LightPaginationNav.svelte'


/home/username/softwares/learn/pokedex-svelte/src/../node_modules/.pnpm/file+..+..+forks+svelte-paginate/node_modules/svelte-paginate/src/main.ts:3:47
Hint: Could not find a declaration file for module './LightPaginationNav.svelte'. '/home/username/softwares/learn/pokedex-svelte/node_modules/.pnpm/file+..+..+forks+svelte-paginate/node_modules/svelte-paginate/src/LightPaginationNav.svelte' implicitly has an 'any' type.
export { default as PaginationNav } from './PaginationNav.svelte'
export { default as LightPaginationNav } from './LightPaginationNav.svelte'
export { default as DarkPaginationNav } from './DarkPaginationNav.svelte'


/home/username/softwares/learn/pokedex-svelte/src/../node_modules/.pnpm/file+..+..+forks+svelte-paginate/node_modules/svelte-paginate/src/main.ts:4:46
Hint: Could not find a declaration file for module './DarkPaginationNav.svelte'. '/home/username/softwares/learn/pokedex-svelte/node_modules/.pnpm/file+..+..+forks+svelte-paginate/node_modules/svelte-paginate/src/DarkPaginationNav.svelte' implicitly has an 'any' type.
export { default as LightPaginationNav } from './LightPaginationNav.svelte'
export { default as DarkPaginationNav } from './DarkPaginationNav.svelte'

```

@bas-baskara, Do you have this issue in your branch when running the `svelte-check`?
I have my own fork https://github.com/azzamsa/svelte-paginate/commits/mine which also contains your changes. But this error pops up.

If we could solve this issue we also able to patch https://github.com/TahaSh/vue-paginate.

**Comment by bas-baskara** [comment]
Hi @azzamsa,
You have to add `"svelte": "src/index.js` and `"type":"module"` into package.json

**Comment by azzamsa** [comment]
I think it is already there https://github.com/azzamsa/svelte-paginate/blob/364affd5527a66df4f69646a53cdf958c46670b2/package.json#L12

**Comment by bas-baskara** [comment]
The result of svelte-check from [https://github.com/bas-baskara/svelte-paginate](url) :

====================================
svelte-check found 0 errors, 0 warnings, and 1 hint
redpoint@redpoint-MS-7D18:~/Documents/GITHUB/spt$ npm run check

> svelte-paginate@0.0.1 check
> svelte-check --tsconfig ./tsconfig.json


====================================
Loading svelte-check in workspace: /home/redpoint/Documents/GITHUB/spt
Getting Svelte diagnostics...

/home/redpoint/Documents/GITHUB/spt/src/generateNavigationOptions.ts:31:11
Hint: 'value' is declared but its value is never read. 
    .fill(null)
    .map((value, index) => ({
      type: 'number',


/home/redpoint/Documents/GITHUB/spt/src/generateNavigationOptions.ts:46:13
Hint: 'value' is declared but its value is never read. 
      .fill(null)
      .map((value, index) => {
        if (index === totalShownPages - 1) {


/home/redpoint/Documents/GITHUB/spt/src/generateNavigationOptions.ts:67:13
Hint: 'value' is declared but its value is never read. 
      .fill(null)
      .map((value, index) => {
        if (index === 0) {


/home/redpoint/Documents/GITHUB/spt/src/generateNavigationOptions.ts:88:13
Hint: 'value' is declared but its value is never read. 
      .fill(null)
      .map((value, index) => {
        if (index === 0) {


/home/redpoint/Documents/GITHUB/spt/src/PaginationNav.svelte:28:31
Hint: Parameter 'option' implicitly has an 'any' type, but a better type may be inferred from usage. (js)

  function handleOptionClick (option) {
    dispatch('setPage', { page: option.value })`
`

**Comment by azzamsa** [comment]
The errors above is produced by the dependant of svelte-paginate.

I am using svelte paginate on my app. And svelte-check produces the errors above.

Do you have the same issue in your app that use svelte-paginate?

**Comment by bas-baskara** [comment]
hi @azzamsa,
Yes but after I modified it by adding "svelte": "src/index.js and "type":"module" into package.json, the error was disappeared.

**Comment by TahaSh** [comment]
This is fixed now in `v0.1.0`.
